x/cron/keeper/genesis.go

```diff
@@ -0,0 +1,36 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/dymensionxyz/rollapp-wasm/x/cron/types"
+)
+
+// InitGenesis initializes the capability module's state from a provided genesis
+// state.
+func (k Keeper) InitGenesis(ctx sdk.Context, genState types.GenesisState) {
+	if err := genState.Validate(); err != nil {
+		panic(err)
+	}
+	var (
+		GameID uint64
+	)
+	// this line is used by starport scaffolding # genesis/module/init
+	for _, item := range genState.WhitelistedContracts {
+		if item.GameId > GameID {
+			GameID = item.GameId
+		}
+		k.SetContract(ctx, item)
```

Comment:
err handle

Reply:
err handling was not necessary here. this is the setter for contract store